superdesk/commands/index_from_mongo.py

```diff
@@ -87,10 +90,17 @@ def get_mongo_items(cls, mongo_collection_name, page_size):
 
         db = app.data.get_mongo_collection(mongo_collection_name)
         args = {"limit": bucket_size, "sort": [(config.ID_FIELD, pymongo.ASCENDING)]}
-        last_id = None
+
         while True:
             if last_id:
+                try:
+                    last_id = ObjectId(last_id)
+                except Exception:
+                    pass
                 args.update({"filter": {config.ID_FIELD: {"$gt": last_id}}})
+                # print last ID for items
+                print(last_id)
```

Comment:
is this temporary?

Reply:
Actually, as per the ticket we have to print` last_id` and I added it here, so we will get the last_id of the items processed every time.

Reply:
not sure we need it every time, would be enough at the end. and pls add an info regarding what resource is it